src/multi_vector_simulator/C0_data_processing.py

```diff
@@ -1727,11 +1727,12 @@ def get_timeseries_multiple_flows(settings, dict_asset, file_name, header):
 
 
 def process_maximum_cap_constraint(dict_values, group, asset, subasset=None):
+    # ToDo: should function be split into separate processing and validation functions?
     """
     Processes the maximumCap constraint depending on its value.
 
     * If MaximumCap not in asset dict: MaximumCap is None
-    * If MaximumCap < installedCap: invalid, MaximumCap is None
+    * If MaximumCap < installedCap: invalid, ValueError raised
```

Comment:
@Bachibouzouk I think we have not completely decided what to use: Did we want to use `ValueError` and so on in the MVS or limit ourselves to `logging.Errors` only, so that we only have one type? What fits with the EPA?

Reply:
The EPA only gets the error message if any is raised, this means that if the simulation fails they will get an error message, but would not necessarily know why it failed because the error will happen during simulation.

Reply:
Okay... so `ValueError` would terminate the simulation, and the user would not know what happened, but if we raise a `logging.Error` this logging message is parsed back to the EPA and the user can change their inputs?

Reply:
The reverse : if an error is raised the MVS simulation is ended here and the error message is meaningful, if you just log it as an error the simulation will surely fail at the solver and throw a generic error "server could not converge" or something like that. 

Only raised errors are directly conveyed to the user in EPA

Reply:
Ohooooo, good to know. Then I guess we should translate some `logging.Errors` to `raise Error` instead thoughout the MVS. 

@ciaradunks okay, then the question is whether this is an `Error `or a `Warning` - do we dare to set the `MAXIMUM_CAP` to a value that seems valid for us (ie. to `INSTALLED_CAP` and thus disable the simulation), or do we say that this is something the user has to decide? While for my personal simulations I prefer a `Warning` I also believe that users might not look at the warnings and thus not realize that a) the simulation is not what they expected b) `MAXIMUM_CAP` does not mean what they think it does. So, I understand why to use an Error.

Reply:
We gather MVS-specific exceptions here: https://github.com/rl-institut/multi-vector-simulator/blob/feature/coupled_maximum_installed_capacities/src/multi_vector_simulator/utils/exceptions.py

Maybe a specific error should be raised for this occasion, ie `MaximumCapValueInvalid(ValueError)`?

Reply:
This could be a nice think to make this specific error

Reply:
```suggestion
    * If MaximumCap < installedCap: invalid, MaximumCapValueInvalid raised
```